tests/storage/snapshot.go

```diff
@@ -100,28 +81,6 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 		})))
 	}
 
```

Comment:
For the sake of review simplicity, please consider having 2 commits: one commit refactors the libstorage, by moving functions from snapshot to the lib. The second commit will introduce the new functionality of successful VM snapshot tracking.

Reply:
why refactor libstorage?

Reply:
it's a bit hard for me at this point but for next time will defiantly do it. is it ok?